authorization-common/src/main/resources/legacyRights.json

```diff
@@ -11,6 +11,7 @@
   "environments/accessEnvironment": "environments/accessEnvironment",
   "environments/repairFreeIPA": "environments/write",
   "environments/adminFreeIPA": "environments/adminFreeIPA",
+  "environments/upgradeFreeIPA": "environments/upgradeFreeIPA",
```

Comment:
value should be `environments/write` I think

Reply:
right!